update zcml to use for attribute rather than layer

lukebrannon opened the pull request:
Fixes: #772 

Improves:
Updates documentation with correct zcml xml.

Changes proposed in this pull request:

- use for attribute in browser:page instead of layer when overriding class view

-

-

lukebrannon committed:
update zcml to use for attribute rather than layer

thet commented:
Defining a layer should be possible for ``browser:view`` as well as ``browser:page`` until zope.components 4.0 (https://github.com/zopefoundation/zope.component/blob/master/CHANGES.rst).
What version of zope.component are you using?

MrTango commented:
https://training.plone.org/5/mastering_plone/views_2.html?highlight=layer#the-default-view

lukebrannon committed:
provide correct zcml for browser override and explanation for marker interface and permissions

lukebrannon committed:
fix note formatting

svx commented on develop/plone/views/browserviews.rst:
```diff
@@ -584,11 +584,18 @@ original.
+Note::
+
+        We've retained the permissions and marker interface of the original view.
+        You may provide a specific permission or marker interface in lieu of these
```
Can we replace *lieu* with *instead of* ?

*Lieu* is not wrong but *instead of* may be easier to understand, we have to keep in mind that our audience is from all over. :)

stevepiercy replied:
What do you have against French? 😜 

"instead of" is more common than the fancy pants "in lieu of", so I would agree.

lukebrannon replied:
@svx updated :)

lukebrannon committed:
Formatting and word choice update to Override note

Correct note format and replace lieu with instead of for audience comprehension.
